site/docs/getting-started/dev.mdx

```diff
@@ -1,11 +1,147 @@
+### 1. Install the Salt packages
+
+There are three dependencies that you need to add to your project:
+
+- `@jpmorganchase/uitk-core` contains production-ready UI components.
+- `@jpmorganchase/uitk-theme` contains CSS files that apply Salt's default theme.
+- `@jpmorganchase/uitk-icons` contains SVG-based icons.
+
+Use this npm command to install all three dependencies in one go:
+
+`npm install --save @jpmorganchase/uitk-core @jpmorganchase/uitk-theme @jpmorganchase/uitk-icons`
+
+If you prefer not to use npm, you can install the packages using yarn or other package managers.
+
+### 2. Add required web fonts
+
+Salt’s default theme requires the Open Sans and PT Mono web fonts. Use one of the following methods to integrate them into your application.
+
+**Use Google Fonts**
+
+To load the fonts directly from Google Fonts, add the following into your application’s index.html file:
+
+```html
+<link rel="preconnect" href="https://fonts.googleapis.com" />
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+<link
+  href="https://fonts.googleapis.com/css2 family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&family=PT+Mono&display=swap"
+  rel="stylesheet"
+/>
+```
+
+For further details and alternative methods, refer to the instructions on Google Fonts tear-out.
+
+**Use FontSource**
+
+You install FontSource’s npm packages and then import their CSS into your application. This will include the necessary web font files into your build so that you can self-host them.
+
+First, install the required NPM packages:
+
+`npm install –save @fontsource/open-sans @fontsource/pt-mono`
+
+Next, import the CSS for the weights and styles that Salt requires. In a typical React project setup, the best place to put these imports is in your src/index.js file:
+
+```ts
+import "@fontsource/open-sans/300.css";
+import "@fontsource/open-sans/300-italic.css";
+import "@fontsource/open-sans/400.css";
+import "@fontsource/open-sans/400-italic.css";
+import "@fontsource/open-sans/500.css";
+import "@fontsource/open-sans/500-italic.css";
+import "@fontsource/open-sans/600.css";
+import "@fontsource/open-sans/600-italic.css";
+import "@fontsource/open-sans/700.css";
+import "@fontsource/open-sans/700-italic.css";
+import "@fontsource/open-sans/800.css";
+import "@fontsource/open-sans/800-italic.css";
+import "@fontsource/pt-mono";
+```
+
+**Other**
+
+If you prefer not to use Google Fonts or FontSource, you can use any other approach to incorporate the required web font files. Make sure the CSS @font-face declarations associate the following combinations of font-family, font-weight and font-style with the corresponding web font files:
+
+TABLE
+
+### 3. Integrate Salt into your app
+
+To integrate Salt into your project, import the CSS theme and ToolkitProvider component—and then wrap your application in it.
+
+For example, in a typical React project, you can do this in your src/index.js as follows:
+
+```ts
```

Comment:
```suggestion
```tsx
```

Reply:
Not sure, if docusarus supports it but generally you would use tsx here

Reply:
yeah that works :)